crates/youki/src/commands/delete.rs

```diff
@@ -1,11 +1,15 @@
-use crate::commands::load_container;
+use crate::commands::{container_exists, load_container};
 use anyhow::{Context, Result};
 use std::path::PathBuf;
 
 use liboci_cli::Delete;
 
 pub fn delete(args: Delete, root_path: PathBuf) -> Result<()> {
```

Comment:
In the `delete()`, the `construct_container_root` is called three times. Can we make `construct_container_root()` execute only a time?

Reply:
Thought about it, but I am only counting one additional allocation which won't made a big difference and it made the code less readable.